components/ui/appHeader.tsx

```diff
@@ -38,9 +42,22 @@ const AppHeader = () => {
           <DropdownMenuTrigger className="focus:outline-none">
             <div className="cursor-pointer">
               <div className="hidden xl:flex">
-                <div className="w-12 h-12 mr-[14px] rounded-full bg-[#321D47] flex items-center justify-center">
-                  <p className="unbound-font text-white font-bold text-xl">{(name || "").charAt(0)}</p>
-                </div>
+                {picture ? (
+                  <div className="w-12 h-12 mr-[14px] rounded-full flex items-center justify-center">
+                    <Image
+                      unoptimized
+                      src={picture}
+                      width={48}
+                      height={48}
+                      alt="Profile Picture"
+                      className="rounded-full"
+                    />
+                  </div>
+                ) : (
+                  <div className="w-12 h-12 mr-[14px] rounded-full bg-[#321D47] flex items-center justify-center">
```

Comment:
Same here with `bg-[#321D47]`. We could translate to something like `--deep-purple: 267 43% 20%;` in HSL, becoming `bg-deep-purple` here